turf/explode/_explode.py

```diff
@@ -0,0 +1,79 @@
+from typing import Dict, List, Sequence, TypeVar
+
+from turf.invariant import get_coords_from_features, get_coords_from_geometry
+
+from turf.helpers import feature, feature_collection, Point
+from turf.helpers import Feature, FeatureCollection, Geometry
+from turf.helpers import all_geometry_types
+
+from turf.utils.helpers import get_input_dimensions
+
+GeoJson = TypeVar("GeoJson", Feature, FeatureCollection, Geometry)
+
+
+def explode(features: GeoJson) -> FeatureCollection:
+    """
+    Takes a feature or set of features and returns all positions as {Point|points}.
+
+    # IMPORTANT! : in the original turf test cases, there were multiple
+    tests with 3 dimensional coordinates [x, y, z]. In the current
+    pyturf implementation only 2 dimensional coordinates are used.
+    Following test cases has been changed towards 2 dimensional coordinates:
+        - geometrycollection-xyz-0-6.json
+        - multilinestring-xyz-0-11.json
+        - multipoint-xyz-0-9.json
+        - multipolygon-xyz-0-10.json
+        - point-xyz-0-8.json
+        - polygon-xyz-0-7.json
+
+    :param features: any GeoJSON feature or feature collection
+    :return: {FeatureCollection} points representing the exploded input features
+    """
+    points = []
+
+    if features.get("type") == "FeatureCollection":
+
+        for feature in features["features"]:
+
+            properties = feature.get("properties", {})
+            coords = get_coords_from_features(feature)
+            points.extend(transform_coordinates_dim(coords, properties))
+
+    elif features.get("type") in "GeometryCollection":
+
+        for geometry in features["geometries"]:
+
+            properties = geometry.get("properties", {})
+            coords = get_coords_from_features(geometry)
+            points.extend(transform_coordinates_dim(coords, properties))
+
+    else:
+
+        properties = features.get("properties", {})
+        coords = get_coords_from_geometry(features)
+        points.extend(transform_coordinates_dim(coords, properties))
+
+    return feature_collection(points)
+
+
+def transform_coordinates_dim(coords: Sequence, properties: Dict) -> Sequence[Feature]:
```

Comment:
Maybe rename this function to `reduce_coordinates_to_points`? or just `transform_coordinates`

Reply:
reduce_coordinates_to_points seems better. 👍